x-pack/plugins/ml/public/application/model_management/test_models/test_flyout.tsx

```diff
@@ -68,6 +53,32 @@ export const TestTrainedModelFlyout: FC<Props> = ({ modelId, onClose }) => {
           </EuiTitle>
         </EuiFlyoutHeader>
         <EuiFlyoutBody>
+          {model.deployment_ids.length > 1 ? (
+            <>
+              <EuiFormRow
+                fullWidth
+                label={
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.testModelsFlyout.deploymentIdLabel"
+                    defaultMessage="Deployment ID"
+                  />
+                }
+              >
+                <EuiSelect
+                  fullWidth
+                  options={model.deployment_ids.map((v) => {
```

Comment:
We should only list started deployments.

Reply:
There are all started

Reply:
Fixed in https://github.com/elastic/kibana/pull/155375/commits/9326a23d25c4b231e92f6ac20cd0e4b39893241d